test/installer/scripts/omsadmin_systest.rb

```diff
@@ -194,11 +202,36 @@ def set_omsagent_proc_limit(val = nil, should_succeed = true)
     return output
   end
 
+  def show_omsagent_proc_limit
+    lstderrprefix='/tmp/show_omsagent_proc_limit'
+    stdout_str = `#{@omsadmin_script} -c 2>#{lstderrprefix}.stderrbuffer`
+    ret_code = $?
+    stderr_str = `cat #{lstderrprefix}.stderrbuffer`
+    `rm -f #{lstderrprefix}.stderrbuffer`
+
+    assert_equal(0, ret_code.to_i, "The command to show the user process limit was unsuccessful")
+    return stdout_str,stderr_str
+  end
+
   def unset_proc_limit
     output = `#{@omsadmin_script} -r`
     ret_code = $?
     assert_equal(0, ret_code.to_i, "The command to unset the proc limit failed")
     return output
   end
 
+  def test_show_omsagent_proc_limit_default
+    set_omsagent_proc_limit()
+    lso, lse = show_omsagent_proc_limit()
+    assert_match(/^-?\d+$/,lso,"The command to show what should be the default proc limit did not yield an integer.")
+    assert_equal(lse, "", "Unexpected error: '#{lse}")
+  end
+
+  def test_show_omsagent_proc_limit_5000
+    set_omsagent_proc_limit(5000)
+    lso, lse = show_omsagent_proc_limit()
+    assert_match(/^5000$/,lso,"Should have been 5000 process limit.  Instead #{lso}.")
+    assert_equal(lse, "", "Unexpected error: '#{lse}")
+  end
+
```

Comment:
Tests you should add:
- showing the limit set in a file with 2+ omsagent lines (you can use `#{@omsadmin_test_dir}/limits-two-settings.conf` found [here](https://github.com/Microsoft/OMS-Agent-for-Linux/blob/master/test/installer/scripts/prep_omsadmin.sh#L73))
- showing the limit when there is no line for omsagent (you can use `#{@omsadmin_test_dir}/limits-no-settings.conf` found [here](https://github.com/Microsoft/OMS-Agent-for-Linux/blob/master/test/installer/scripts/prep_omsadmin.sh#L83))

Reply:
Ok.